content/en/guides/core/registry/configure_registry.md

```diff
@@ -107,15 +118,49 @@ The proceeding table demonstrates the highest level of permission in the event o
 | Team registry role | Individual registry role | Inherited registry role |
 | ------ | ------ | ------ | 
 | Viewer | Viewer | Viewer |
+| Viewer | Restricted Viewer | Viewer |
 | Member | Viewer | Member |
-| Admin  | Viewer | Admin  | 
+| Member | Restricted Viewer | Member |
+| Admin  | Viewer | Admin  |
+| Admin  | Restricted Viewer | Admin  | 
 
 If there is a conflict, W&B displays the highest level of permissions next to the name of the user.
 
 For example, in the proceeding image Alex inherits **Member** role privileges because they are a member of the `smle-reg-team-1` team.
 
 {{< img src="/images/registry/role_conflict.png" alt="Registry role conflict resolution" >}}
 
+### Registry ownership
+
+Registry admins can assign a **Restricted Viewer** user as the registry's owner. Registry ownership is primarily for accountability purposes and does not confer any additional permissions beyond those granted by the user's assigned role. A Restricted Viewer who is also the registry owner will still have only read-only access to the registry content.
+
+### SDK compatibility
+
+{{% alert title="SDK version requirement" %}}
+To use the Restricted Viewer role, you must use W&B SDK version 0.19.9 or higher. Older SDK versions will receive permission errors when trying to access artifacts in registries where users have Restricted Viewer roles.
+{{% /alert %}}
+
+The Restricted Viewer role affects SDK behavior in the following ways:
```

Comment:
Address all of this feedback systematically for the entries in this section. For example, adjust all of the entries that allow for viewing metadata only to align with the feedback in line 149.